### ISSUE TITLE
Question: Password fields on registration form

### ISSUE THREAD

**Issue by unicornis-pl** [jfarthing84/theme-my-login#18]
Is there a way to get similar fields and functionality (password strenght indicator, confirmation of weak password usage)?

I am using custom forms, I copied a table "Account Management" from profile-form.php to register-form.php, fields appear, but all the fields are inactive.

**Comment by jfarthing84** [comment]
You would also need to load the user profile javascript file.

**Comment by unicornis-pl** [comment]
Thanks, I am "almost there". 
I enabled js by wp_enqueue_script( 'user-profile' );  Is this what you meant?
Two more questions:
1. How to prevent users from using weak passwords? The strength meter works fine, the "confirm usage of weak password" check box appears, but I cannot figure out how to block usage of weak passwords, as even weak passwords are accepted. I see that on profile form the submit button gets disabled but I cannot pin-point where...
2. The same applies to resetting lost passwords. It accepts even single character passwords. I checked it on vanilla install, so it is not theme related.
